tests/follow.test.js

```diff
@@ -41,28 +40,31 @@ describe('TEST USER FOLLOW', () => {
       });
   });
 
-  it('It should return a 201 if a user is been followed successfully', done => {
+  it('It should return a 201 if a user followed successfully', done => {
     chai
       .request(app)
       .post('/api/v1/follow/6517a6ea-662b-4eef-ab9f-20f89bd7099c')
       .set('authorization', token1)
       .end((err, res) => {
         expect(res.status).to.equal(201);
         expect(res.body.message).to.be.a('string');
-        expect(res.body.message).to.equal('Successfully followed user');
+        expect(res.body.message).to.equal(
+          'You have Successfully followed user'
+        );
         done();
       });
   });
 
-  it('It should return a 403 if you are already following a user', done => {
+  it('It should return a 403 if try to follow twice', done => {
```

Comment:
Same as ☝️ also I think the former test description was ok

Reply:
Noted and fixed